koji/vars/external_repos.yml

```diff
@@ -1,61 +1,31 @@
 ---
 external_repos:
-  - name: puppetlabs-pc1-fedora-24
-    url: file:///mnt/koji/external-repos/www/puppetlabsf24-$arch/RPMS.pc1/
   - name: rhel-7.0-server
     url: file:///mnt/koji/external-repos/rhel7-$arch/7.0/
   - name: epel-7
     url: file:///mnt/koji/external-repos/www/epel7-$arch/RPMS.epel/
-  - name: fedora-23
-    url: file:///mnt/koji/external-repos/www/fedora23-$arch/RPMS.os/
-  - name: fedora-23-updates
-    url: file:///mnt/koji/external-repos/www/fedora23-$arch/RPMS.updates/
   - name: centos-sclo-rh-rhel-7
     url: file:///mnt/koji/external-repos/www/centos7-$arch/RPMS.sclo.rh/
   - name: centos-sclo-rh-rhel-6
     url: file:///mnt/koji/external-repos/www/centos6-$arch/RPMS.sclo.rh/
   - name: rhscl-1.0
     url: file:///mnt/koji/external-repos/rhscl-rhel6-$arch/1.0/
-  - name: fedora-20
-    url: file:///mnt/koji/external-repos/www/fedora20-$arch/RPMS.os/
-  - name: fedora-20-updates
-    url: file:///mnt/koji/external-repos/www/fedora20-$arch/RPMS.updates/
   - name: rhel-6.5-server
     url: file:///mnt/koji/external-repos/rhel6-$arch/6.5/
   - name: rhel-6.5-server-optional
     url: file:///mnt/koji/external-repos/rhel6-$arch/6.5-optional/
-  - name: fedora-22
-    url: file:///mnt/koji/external-repos/www/fedora22-$arch/RPMS.os/
   - name: epel-7-beta
```

Comment:
Why do we have epel-7-beta? Do we ever use this?

Reply:
Any idea about this?

Reply:
Probably way back in the day we did. I am only not removing it here because I am performing a sequence of removals, with this round being old Fedora. Then I plan to tackle the EL landscape.